auto_tune_vllm/execution/trial_controller.py

```
@@ -0,0 +1,591 @@
+                    # Log accelerator types if available
+                    accelerator_types = [k for k in cluster_resources.keys() if k.startswith("accelerator_type:")]
+                    if accelerator_types:
+                        for acc_type in accelerator_types:
+                            acc_name = acc_type.replace("accelerator_type:", "")
+                            acc_count = cluster_resources[acc_type]
+                            logger.info(f"GPU type: {acc_name} (count: {acc_count})")
+                else:
+                    logger.warning("No GPUs detected in Ray cluster. vLLM may fail to start.")
+            else:
+                logger.warning("Ray not initialized. Cannot check GPU availability.")
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)